### PR TITLE
Added build script for crass

### PR DESCRIPTION
**Pull request by vathsalashetty25**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by vathsalashetty25** [comment]
Closing PR as Travis did not get triggered.

**Comment by vathsalashetty25** [comment]
Reopening PR to trigger Travis job.

File: c/crass/crass_rhel_8.4.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package               : crass
+# Version               : v1.0.4
+# Source repo           : https://github.com/rgrove/crass
+# Tested on             : UBI 8.5
```
Comment (seth-priya) [inline c/crass/crass_rhel_8.4.sh:7]: discrepancy in script name and this line

File: c/crass/crass_rhel_8.4.sh [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+if [ -d "crass" ] ; then
+  rm -rf crass
+fi
+
+
+yum install -y git ruby procps yum-utils wget
+
+yum-config-manager --add-repo http://mirror.centos.org/centos/8/AppStream/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/PowerTools/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/virt/ppc64le/ovirt-44/
+
+wget https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official && mv RPM-GPG-KEY-CentOS-Official /etc/pki/rpm-gpg/. && rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
```
Comment (seth-priya) [inline c/crass/crass_rhel_8.4.sh:39]: see if you can do w/o installing/enabling centos repos

File: c/crass/crass_rhel_8.4.sh [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+#
+# Disclaimer            : This script has been tested in root mode on given
+# ==========              platform using the mentioned version of the package.
+#                         It may not work as expected with newer versions of the
+#                         package and/or distribution. In such case, please
+#                         contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
```
Comment (seth-priya) [inline c/crass/crass_rhel_8.4.sh:21]: this is reduntant

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: c/crass/crass_rhel_8.4.sh (hunk 2 above)
Comment (vathsalashetty25) [reply c/crass/crass_rhel_8.4.sh]: Yes..i have updated the script w/o installing/enabling centos repos..and it's working fine

**Comment by seth-priya** [comment]
@vathsalashetty25 the travis build for this seems to be failing due to missing dependencies, please check and update the script

**Comment by vathsalashetty25** [comment]
updated the script..Travis check passing

**Review by seth-priya — APPROVED** [review]

**Pull request merged**